docs/build/build-storage.md

```diff
@@ -71,9 +75,13 @@ built on the key-value store.
 The key-value data store is backed by [RocksDB](https://rocksdb.org/), and it also supports an 
 experimental [Parity database](https://github.com/paritytech/parity-db).
 
-> The database is used for components that require persistent storage: Substrate clients, Substrate light-clients
-> & off-chain workers. For more information, check out
-> Substrate Developer Hub's [Storage Page](https://docs.substrate.io/v3/advanced/storage/).
+:::note The database is used for components that require persistent storage
+
+These are components like Substrate clients, Substrate light-clients & off-chain workers. 
```

Comment:
This line is not reading well. Probably remove the content from the title and use the actual content inside the note